.github/actions/fastlane/action.yaml

```diff
@@ -0,0 +1,82 @@
+name: 'fastlane'
+description: 'Runs fastlane with target'
+inputs:
+  environment:
+    description: "According .env file provided in fastlane folder, usually one of [local, dev, staging, prod]"
+    required: true
+  platform:
+    description: "Build platform declared in Fastfile, usually one of [android, ios]"
+    reauired: true
+  target:
+    description: "Build target declared in Fastfile, usually one of [build, release]"
+    required: true
+  # Optional
+  ruby_version:
+    required: false
+    default: '3.0'
+  bundler_version:
+    description: Option for 'gem install` command
+    required: false
+    default: '2.2.16'
+  # Android
+  keystore:
+    description: "keystore for android sign"
+    required: false
+    default: ""
+  keystore_pass: 
+    description: "Password for android keystore"
+    required: false
+    default: ""
+  java_distribution:
+    description: Option of action/setup-java@v2
+    required: false
+    default: 'adopt'
+  java_version:
+    description: Option of action/setup-java@v2
+    required: false
+    default: '11'
+  # iOS
+  certificate:
+    description: "Certificate for ios sign"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - name: Setup ruby
+      if: ${{ runner.name == 'Hosted Agent' }}
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ inputs.ruby_version }}
+        bundler-cache: true
+    - name: Setup java
+      if: ${{ inputs.platform == 'android' }}
+      uses: actions/setup-java@v2
+      with:
+        distribution: ${{ inputs.java_distribution }}
+        java-version: ${{ inputs.java_version }}
+    - name: Setup dependencies
+      # Setup dependencies for fastlane package from Gemfile
+      shell: bash
+      run: |
+        gem install bundler:${{ inputs.bundler_version }}
+        bundle install --jobs 4 --retry 3
+    - name: Update ci keychain for iOS
+      # "Populates 'ci' keychain with provided external cert for code sign, see more: https://blog.codemagic.io/how-to-code-sign-publish-ios-apps/"
+      if: ${{ inputs.platform == 'ios' && inputs.certificate != ''}}
```

Comment:
let's avoid these IFs, in order to avoid them - we should have separate yaml files one for ios and one for android.

Reply:
Done